### PR TITLE
Pb2 create db scheme

### PR DESCRIPTION
**Pull request by FedosovMax**

### CODE REVIEW

File: create-tables.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+--
+
+DROP TABLE IF EXISTS `phone_book_item`;
+CREATE TABLE `phone_book_item` (
+  `name` varchar(45) NOT NULL,
+  `surname` varchar(45) NOT NULL,
+  `mobile phone` int(55) NOT NULL,
+  `home phone (optional)` int(55) DEFAULT NULL,
+  `address (optional)` varchar(45) DEFAULT NULL,
+  `email (optional)` varchar(45) DEFAULT NULL,
```
Comment (bohdanvan) [inline create-tables.sql]: *mobile phone*, *home phone (optional)*, *address (optional)*, *email (optional)* are very odd names.
Do not use ' ', '(', ')' in names.
Please, replace them with *mobile_phone*, *address* etc.

File: create-tables.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
+--
+-- Table structure for table `phone_book_item`
+--
+
+DROP TABLE IF EXISTS `phone_book_item`;
+CREATE TABLE `phone_book_item` (
+  `name` varchar(45) NOT NULL,
+  `surname` varchar(45) NOT NULL,
+  `mobile phone` int(55) NOT NULL,
```
Comment (bohdanvan) [inline create-tables.sql]: Use *varchar* type for *mobile phone*.

File: create-tables.sql [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+CREATE TABLE `phone_book_item` (
+  `name` varchar(45) NOT NULL,
+  `surname` varchar(45) NOT NULL,
+  `mobile phone` int(55) NOT NULL,
+  `home phone (optional)` int(55) DEFAULT NULL,
+  `address (optional)` varchar(45) DEFAULT NULL,
+  `email (optional)` varchar(45) DEFAULT NULL,
+  `users_name` varchar(45) NOT NULL,
+  PRIMARY KEY (`name`),
+  UNIQUE KEY `name_UNIQUE` (`name`),
```
Comment (bohdanvan) [inline create-tables.sql]: Different users can have their own phone book items with the same key.
In your case, if Taras adds item with name *My wife*, Ivan won't be able to add item *My wife*.

Please, provide separate *id* field for the purpose of primary key.

File: create-tables.sql [hunk 4]
```diff
@@ -0,0 +1,31 @@
+--
+-- Table structure for table `phone_book_item`
+--
+
+DROP TABLE IF EXISTS `phone_book_item`;
+CREATE TABLE `phone_book_item` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `surname` varchar(45) DEFAULT NULL,
+  `mobile phone` varchar(45) NOT NULL,
```
Comment (bohdanvan) [inline create-tables.sql]: 'mobile phone' -> 'mobile_phone'

**Comment by bohdanvan** [comment]
Please, move this 2 scripts into '/sql' directory inside the project.

File: create-tables.sql (hunk 4 above)
Comment (FedosovMax) [reply create-tables.sql]: I've seen only this sql script and ran it. Must be another one?

**Pull request merged**